cmd/up/activate.go

```diff
@@ -402,6 +402,9 @@ func (up *upContext) waitUntilDevelopmentContainerIsRunning(ctx context.Context,
 				}
 				continue
 			}
+			if up.Pod.UID != pod.UID {
```

Comment:
Can we do this in the watch events?
We should ignore events from other pods in the `event := <-watcherEvents.ResultChan()` case too